### PR TITLE
[SuggestionsList] props.target to match Callout

### PR DESCRIPTION
**Pull request by richardtagger**
Pass through prop type def should match underlying Component's

## Pull request checklist

* [x] Component `README.md` file is up-to-date.
* [x] Component is unit tested.

### CODE REVIEW

**Comment by richardtagger** [comment]
@swese44 @unindented

File: src/components/SuggestionsList/SuggestionsList.tsx [hunk 1]
```diff
@@ -8,12 +8,13 @@ import Text, { TextSize, TextColor } from '../Text';
...
 import Block, { GutterSize } from '../Block';
 import './SuggestionsList.css';
+import { IPoint } from 'office-ui-fabric-react/lib/utilities/positioning';
 
 export interface SuggestionsListProps extends NestableBaseComponentProps {
   /**
    * The target that the Callout should try to position itself based on.
    */
-  target: HTMLElement;
+  target?:  HTMLElement | string | MouseEvent | IPoint | null;
```
Comment (unindented) [inline src/components/SuggestionsList/SuggestionsList.tsx]: Why is it optional and `null`? Where will it render when it's `undefined` or `null`?

Comment (richardtagger) [reply src/components/SuggestionsList/SuggestionsList.tsx]: Just aligned it to Fabrics. The only one I intend to use is IPoint

Comment (unindented) [reply src/components/SuggestionsList/SuggestionsList.tsx]: Right, but what's the effect of `undefined` and `null`? If it doesn't make sense in our scenario, no sense in aligning.

Comment (richardtagger) [reply src/components/SuggestionsList/SuggestionsList.tsx]: ok done

**Pull request merged**